src/com/aca/homework/week4/snake/Snake.java

```diff
@@ -0,0 +1,146 @@
+package com.aca.homework.week4.snake;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Snake {
+    private char[][] board;
+    private int[][] snake;
+    private int snakeSize = 1;
+    private int appleRow;
+    private int appleCol;
+    private boolean running = true; // if the player loses, the variable becomes false
+
+    public Snake(char[][] board, int headRow, int headCol, int appleRow, int appleCol) {
```

Comment:
you could initialize your board in the constructor instead of requiring it from others

Reply:
same for all other fields